docs/admin/install.rst

```diff
@@ -337,6 +340,56 @@ To start the SecureDrop Workstation client, double-click the SecureDrop desktop
 
 Once the update check is complete, the client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
 
+Troubleshooting installation errors
+-----------------------------------
+
+"Recurse failed: none of the specified sources were found"
```

Comment:
Note that this is expected to be resolved by https://github.com/freedomofpress/securedrop-workstation/pull/530 but until that makes its way into a release (and we've verified that the issue no longer occurs) it makes sense to document this.